salt/modules/dsc.py

```diff
@@ -194,3 +194,69 @@ def remove(name):
     cmd = 'Uninstall-Module {0}'.format(name)
     no_ret = _pshell(cmd)
     return name not in list_modules()
+
+
+def run_config(path):
+    # Run the DSC Configuration
+    cmd = '$job = Start-DscConfiguration -Path {0};'.format(path)
+    cmd += 'Do{ } While ($job.State -notin \'Completed\', \'Failed\'); ' \
+           'return $job.State'
+    ret = __salt__['cmd.run'](cmd, shell='powershell', python_shell=True)
```

Comment:
We certainly accept non-production-ready code into the develop branch, which has never been guaranteed to be stable. That said, it's generally a good practice to document experimental code as-such to ensure that it isn't accidentally used by anybody and to create an issue to track the readiness of the code. Obviously, the last thing we want to do is to cut a release with code in that is not ready to go.